redi-connect-front/src/pages/front/signup/SignUpLanding.tsx

```diff
@@ -3,7 +3,7 @@ import { Link } from 'react-router-dom'
 import { Columns } from 'react-bulma-components'
 import { LoggedOutLayout } from '../../../layouts/LoggedOutLayout'
 
-export default function SignUpLanding () {
+export default function SignUpLanding() {
```

Comment:
huh, this means the formatter is not working properly... oh noooo....

Reply:
I might did that manually but I am not sure ... let's see if it comes up again